### DISCUSSION TITLE
Why does this blog have so many ads?

### DISCUSSION THREAD

**Discussion by sullivan-backend** [devbattery/won4885.github.io-branch-error#10] [Q&A] [answered]
-_-

**Comment by devbattery** [comment, marked as answer]
According to [Smart Blogger](https://web.archive.org/web/20190401011318/https://smartblogger.com/blog-ads/):

“The average “click-through rate” (CTR, or times that a reader clicks on an ad) varies by industry; however, the average is around 0.1%. Let’s say the average cost-per-click (how much you earn when somebody clicks an ad) is $0.50. That means you would need around 1,000 unique visitors each day to get even one ad click and earn $0.50 for the day.

If you enjoy 1,000 unique visitors each day to your blog, you can do a lot better than $0.50 per day in income. The bottom line is that advertising requires a ton of traffic to be worthwhile.”

Basically, the more ads, the more chances you give a reader to engage with your advertising.

And certain ads pay more than others. For instance, that really annoying video ad that starts playing when your mouse hovers over it, that ad pays more than the one just sitting casually over on my sidebar.

Or the ad that immediately follows every single paragraph. (or before) Those tend to have higher engagement because they fit “seamlessly” into the blog post. (I was going to add a picture to illustrate this, but I am assuming there will be an ad right above this so I don’t really need a picture!)

The point being… I don’t run ads to be greedy. This is one of the most reliable (but unreliable-my income changes by as much as 60% month to month!!) ways I have of earning an income for the work I do. Just a few days ago my husband quit his job and at this point, I am the sole income earner for my family. This is a great responsibility and one that I do not take lightly, always considering the future of my 5 kids and home. It’s my job.

To everyone who hates all these ads and never wants to visit this blog again and who thinks I should be jailed (yes, one angry woman thought I should be prosecuted and jailed for excessive advertising), I am sorry.

I hope and pray that you will forgive the intrusion and still opt to hang out here. It wouldn’t be the same without you!

- [Reference](https://iambaker.net/food-blog-advertising/)
